api-server/web-server/src/api/json_helpers.rs

```diff
@@ -185,6 +185,14 @@ pub fn txoutput_to_json(
                 "destination": Address::new(chain_config, dest.clone()).expect("no error").as_str(),
             })
         }
+        TxOutput::AnyoneCanTake(data) => {
+            json!({
+                "type": "AnyoneCanTake",
+                "withdraw_key": Address::new(chain_config, data.withdraw_key().clone()).expect("no error").as_str(),
+                "ask_value": outputvalue_to_json(data.ask(), chain_config, token_decimals),
+                "give_value": outputvalue_to_json(data.give(), chain_config, token_decimals),
+            })
+        }
     }
```

Comment:
If this PR to go through, this should be handled. We cannot keep todo!() like this because it will crash the API server when used. The API server is used by exchanges.

Reply:
I was on a wrong tip. Resolving.